extern/base64_encode/base64_encode.hpp

```diff
@@ -0,0 +1,71 @@
+/*
+* Base64 encoding/decoding (RFC1341)
```

Comment:
We already have `modp_b64` and it can do encoding, too.

Reply:
Swapped base64_encode.hpp with modp_b64